src/libs/highlevel/README.md

```diff
@@ -9,7 +9,7 @@ should be hard to use it in a wrong way. This tutorial gives an introduction for
 using the high-level API.
 
 The API supports all CORBA Basic Data Types, except for `wchar`, as well as the `string` type
-(see also [Data Types](../high-level-api#data-types) below).
+(see also [Data Types](#data-types) below).
```

Comment:
This does not work on the homepage (also not the previous version)

https://www.libelektra.org/tutorials/high-level-api